tests/test_datasets/test_dataset.py

```diff
@@ -219,6 +219,12 @@ def test_get_data_corrupt_pickle(self):
         self.assertIsInstance(xy, pd.DataFrame)
         self.assertEqual(xy.shape, (150, 5))
 
+    def test_equality_comparison(self):
+        # verify the implemented comparison of the dataset object
```

Comment:
Typically comments should explain the *why* not *what*. Just from reading the code it's easy to tell what this test is testing. (also the *why* is obvious, so I think it would have been better to leave the comment out altogether)